src/js/app.js

```diff
@@ -9,7 +9,35 @@
    */
   ns.app = {
 
-    init : function () {
+    init: function () {
+      /**
+        * i18n which stands for internalization is being applied to Piskel to allow for all content in Piskel
+        * to be translated
+        *
+        * Currently there is a directory called i18n which contains the locale directory. In this directory, you
+        * will find all the string translations in JSON format. Each file is named as follows: languageCode_countryCode
+        * For example: en_us.json where en is English and us is United States
```

Comment:
Try to avoid saying "Currently" because that kind of implies that there is a plan to change it in the future. See my comments in `README.md` for my suggested changes to this paragraph.